metadata-ingestion/src/datahub/ingestion/graph/client.py

```diff
@@ -549,8 +549,10 @@ def get_aspects_for_entity(
 
         return result
 
-    def get_entity_semityped(self, entity_urn: str) -> AspectBag:
-        """Get all non-timeseries aspects for an entity (experimental).
+    def get_entity_semityped(
+        self, entity_urn: str, aspects: Optional[List[str]] = None
+    ) -> AspectBag:
+        """Get (all) non-timeseries aspects for an entity (experimental).
```

Comment:
might as well remove the experimental tag on this, given it's pretty stable at this point

```suggestion
        """Get (all) non-timeseries aspects for an entity.
```